Resources/Prototypes/Reagents/Materials/materials.yml

```diff
@@ -91,3 +91,11 @@
   icon: { sprite: Objects/Materials/silk.rsi, state: icon }
   color: "#eeeeee" #eeeeeeeeeeeeeeeeeeeeeeeeeeeeeee
   price: 0 # Maybe better for it to be priceless, knowing how greedy cargo is.
+
+- type: material
+  id: Bones
+  name: materials-bones
+  unit: materials-unit-bones
```

Comment:
unit means that in protolate menu it would be shown as planks of wood, bars of gold, sheets of steel, and bones - bones of bones...
so maybe write here piece or bunch

Reply:
done

Reply:
might be worth supporting null units, where its just X material since 5 bones is a lil better than 5 pieces of bone

Reply:
sadly, out of scope of PR. Can try to do this later.